src/Symfony/Bridge/Doctrine/Form/ChoiceList/DoctrineChoiceLoader.php

```diff
@@ -26,6 +26,8 @@ class DoctrineChoiceLoader implements ChoiceLoaderInterface
 {
     /**
      * @var ChoiceListFactoryInterface
+     *
+     * @deprecated since 3.1 to be removed in 4.0.
```

Comment:
Alright! I thought as it is just deprecated to pass it in the constructor, it may be forgotten to be removed in 4.0.

Should we keep a simple comment like `BC, to be removed in 4.0.` or there's no need too?